lib/elixir/lib/supervisor.ex

```diff
@@ -720,10 +744,14 @@ defmodule Supervisor do
   @doc """
   Builds and overrides a child specification.
 
-  Similar to `start_link/2` and `init/2`, it expects a
-  `module`, `{module, arg}` or a map as the child specification.
-  If a module is given, the specification is retrieved by calling
-  `module.child_spec(arg)`.
+  Similar to `start_link/2` and `init/2`, it expects a module, `{module, arg}`
+  or a child specification.
```

Comment:
Either we link here to `t:child_spec/0`, or we keep the wording "map child specification" in there. 🙃

Reply:
fixed.